decidim-core/config/locales/es.yml

```diff
@@ -202,6 +202,7 @@ es:
     newsletter_mailer:
       newsletter:
         note: Has recibido este mensaje porque estás suscrito a boletines de noticias en %{organization_name}. Puedes cambiar la configuración en tu <a href="%{link}">página de notificaciones</a>.
+        see_on_website: ¿No puedes ver este correo electrónico correctamente? Ver en el <a href="%{link}"> sitio web </a>.
```

Comment:
Please, add locales for only English. Translations are handled via Crowdin, and if we add some translations manually directly to the code the integration gets weird.